tasks/main.yml

```diff
@@ -7,15 +7,20 @@
   sudo: yes
   apt: pkg=apache2 state=latest
 
-- name: Install Apache Modules
-  apache2_module: state=present name={{ item }}
-  notify: restart apache
-  with_items:
-    - rewrite
-    - vhost_alias
-    - headers
-    - expires
-    - filter
+- name: Install Apache Modules rewrite
```

Comment:
@naxhh, I'll do it, but I'm trying to know why it fails only on TraveisCi environment, because in my local vagrant instalation (Ubuntu 12.04 and 14.04) and on AWS (12.04 and 14.04) it works fine!